llvm/test/ThinLTO/X86/Inputs/operator-new.ll

```diff
@@ -0,0 +1,11 @@
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
+
+declare ptr @malloc(i64)
+
+define ptr @_ZnwmRKSt9nothrow_t(i64 %size, ptr %0) {
```

Comment:
Is this separate input file needed for the test? Why can't this function be in the main input file, or even have the call to malloc inlined into DoStuff to start with?

Reply:
Originally it was in separate source file (as part of larger C++ program), but as we don't run optimizations on source IR this "operator new" won't be converted to calloc by opt, so we can safely move it to main input file. I'll update the patch.